ironic.conf.j2

```diff
@@ -1,5 +1,10 @@
 [DEFAULT]
+{% if "HTTP_BASIC_HTPASSWD" in env %}
```

Comment:
Should we check that the value isn't empty?

Reply:
good point. I'll update.